lib/Components/Cart/cart.css

```diff
@@ -0,0 +1,170 @@
+    min-width: 100vh;
+    font-size: 1.6rem;
+    font-weight: 500;
+    color: var(--COLOR-FONT);
+    background: var(--COLOR-BG);
+    display: grid;
+    grid-template-areas:
+        "header"
+        "main"
+        "footer";
+    grid-template-columns: auto;
+    grid-template-rows: 7rem 1fr 7rem;
+    margin: auto;
+    
+}
+
+/* MAIN STARTS */
+.sm-main-cart{
+    grid-area: main;
+    min-width: 100%;
+    height: 100%;
+    padding: 2rem;
+    display: grid;
+    grid-template-columns: 1fr 30rem;
+}
+
+.sm-cart-products-list{
+    display: flex;
+    justify-content: flex-start;
+    gap:1rem;
+    flex-wrap: wrap;
+    height: fit-content;
+}
+
+.sm-cart-products-list .sm-card{
+    width:90%;
+}
+
+.sm-cart-products-list .sm-card .sm-card-img{
+    background: var(--COLOR-WHITE-ALWAYS);
+    padding: 2rem;
+}
+
+
+.sm-cart-products-list .sm-card .sm-card-content{
+    padding: 5px;
+    padding-left: 1rem;
+}
+
+.sm-cart-products-list .sm-card .sm-card-content .sm-card-content-desc{
+ font-size: 1.2rem;
+ padding: 1px;
+}
+
+.sm-card-cart-qty-container{
+    display: flex;
+    align-items: center;
+    gap:1rem;
+}
+
+.sm-cart-qty{
+    display: flex;
+    align-items: center;
+    gap: 1rem;
+    font-size: 1.6rem;
+}
+
+.sm-qty-counter-wrapper{
+    display: flex;
+    align-items: center;
+    gap: 8px;
+    border-radius: 5px;
+    border: 2px solid var(--COLOR-PRIMARY);
+}
+
+.sm-cart-qty select{
+    border-radius: 5px;
+    padding: 6px;
+    background: var(--COLOR-BG);
+    color: var(--COLOR-FONT);
+    border: 2px solid var(--COLOR-PRIMARY);
+    outline: none;
+    cursor: pointer;
+    font-weight: 800;
+    transition: all ease 0.2s;
+}
+
+.sm-cart-qty select:hover{
+    background: var(--COLOR-PRIMARY);
+}
+
+.qty-btn-red{
+    color: var(--COLOR-FONT);
+    background: var(--COLOR-RED-60);
+    border: 2px solid var(--COLOR-RED);
+    border-radius: 5px;
+    padding: 5px 10px;
+    white-space: nowrap;
+    cursor: pointer;
+}
+.qty-btn-red:hover{
+    background: var(--COLOR-RED);
+}
+.qty-btn-red:active{
+    transform: scale(0.98);
+}
+
+.sm-cart-checkout-container{
+    display: flex;
+    flex-direction: column;
+    padding: 1.2rem 2rem;
+    border-radius: 5px;
+    border: 1px solid var(--COLOR-PRIMARY);
+    height: fit-content;
+    box-shadow: 0px 3px 2px -2px var(--COLOR-PRIMARY);
+}
+
+.sm-cart-checkout-title{
+    font-weight: 800;
+    font-size: 2.5rem;
+    padding-bottom: 1rem;
+    text-align: center;
+    border-bottom: 1px solid var(--COLOR-PRIMARY);
+}
+
+.sm-cart-checkout-details{
+    display: flex;
+    justify-content: space-between;
+    padding-top: 1rem;
+    font-weight: 500;
+    font-size: 2rem;
+}
+.sm-cart-checkout-amt{
+    font-weight: 700;
+}
+.sm-cart-total{
+    border-top: 1px solid var(--COLOR-PRIMARY);
+    margin: 1rem 0;
+}
+
+.btn-place-order{
+    font-size: 2rem;
+}
+/* MAIN ENDS */
+
+/* MEDIA QUERIES */
+@media only screen and (max-width:800px){
+    :root{
```

Comment:
Can replace the max-width values with variables, so that it would be easy to manage and change them at one place.